src/App.js

```diff
@@ -1,15 +1,94 @@
 import React, { Component } from 'react'
 import ListArticles from './Components/ListArticles'
+import LoginForm from './Components/LoginForm'
+import { signInUser } from './state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
 
 class App extends Component {
+  state = {
+    renderLoginForm: false,
+    email: '',
+    password: '',
+    getArticlesIndex: true
+  }
+
+  renderArticleGetState = () => {
```

Comment:
We do not need this anymore, right? Please remove.

Reply:
Why would you need a closing tag that has no corresponding opening tag?
```
        <**Input** iconPosition='left' placeholder="Email" id="email-input" name="email" onChange={props.inputChangeHandlerProps}>
          <Icon name='at' />
          <input />
        **</Input>**
        **<Input** id="password-input" placeholder="Password" name="password" onChange={props.inputChangeHandlerProps}>
          <input />
        **</Input>**
```

Reply:
Because if you remove the <input /> then there is not input field on the page. I've tired it out on the semantic-ui-react website.

Reply:
That is really strange...on the cooper challenge, why wasn't that an issue?

```
          <Input
            label={{ basic: true, content: 'Years' }}
            labelPosition='right'
            placeholder='Enter your age...'
            id="age" onChange={props.inputChangeHandler}
          />
```